### DISCUSSION TITLE
Release frequency?

### DISCUSSION THREAD

**Discussion by Lyeeedar** [visgl/deck.gl#7230] [Q&A] [answered]
Im looking to use the latest code merged into the main branch, how long do you estimate it will be before a new release is done?

**Comment by Pessimistress** [comment, marked as answer]
#7205 is included in v8.9.0-alpha.2. The alpha track does not currently contain any other unstable changes. If you want it in a patch, we will need some more testing.

**Reply by Lyeeedar** [reply]
Oh, I didn't realise that there are alpha builds I can use. I shall try that, thank you.
